man/shard.yml.5

```diff
@@ -524,7 +524,8 @@ A path to the source file to compile (string).
 .SH "DEPENDENCY ATTRIBUTES"
 .sp
 Each dependency needs at least one attribute that defines the resolver for this
-dependency. Those can be \fIpath\fP, \fIgit\fP, \fIgithub\fP, \fIgitlab\fP, \fIbitbucket\fP.
+dependency. Those can be \fIpath\fP, \fIgit\fP, \fIgithub\fP, \fIgitlab\fP, \fIbitbucket\fP,
+\fIcodeberg\fP.
```

Comment:
```suggestion
dependency. Those can be \fIpath\fP, \fIgit\fP, \fIgithub\fP, \fIgitlab\fP, \fIbitbucket\fP, \fIcodeberg\fP.
```

Reply:
The man pages should be generated with `make manpages` and contain whatever the generator produces.

Reply:
My bad, I am going to fix in 2 hours.

Reply:
Updated the man pages.